### PR TITLE
graphql: Record queries that trigger getCommit calls

### PR DESCRIPTION
**Pull request by tsenart**
This is an attempt to debug #15392 and find what GraphQL queries trigger
the `getCommit` codepath, which leads to a spike of git log calls in git
server, that triggers that alert.

In the future, we should invest in working tracing, rather than only
per-request activated tracing, which doesn't allow us to retroactively
debug situations like these.

### CODE REVIEW

**Comment by tsenart** [comment]
@keegancsmith: Decided to not reset the histogram every 5 minutes, and instead just log the current state at that interval.

File: internal/vcs/git/commits.go [hunk 1]
```diff
@@ -60,12 +64,31 @@ type CommitsOptions struct {
...
 		return Mocks.GetCommit(id)
 	}
 
+	if recordQueries {
+		q, _ := ctx.Value("graphql-query").(string)
+		queries.Add(q, 1)
+
+		select {
+		case <-flushTicker.C:
+			log15.Error("getCommit", "queries", queries.String())
```
Comment (unknwon) [inline internal/vcs/git/commits.go:87]: Not sure why it is an error log 

```suggestion
			log15.Info("getCommit", "queries", queries.String())
```

**Review by unknwon — APPROVED** [review]

**Comment by tsenart** [comment]
Decided to use Honeycomb to debug this instead.

**Review by keegancsmith — APPROVED** [review]
nice, using honeycomb is a great idea.

**Pull request merged**